Phonebook.Frontend/src/app/pages/page-information/page-information.component.html

```diff
@@ -33,10 +33,35 @@ <h1 i18n="PageInformationComponent|Title of the page information@@PageInformatio
         <span>
           You've found a bug or have an idea for a new feature?
         </span>
+      </mat-card-content>
+      <mat-card-footer>
         <span class="pb-center-element">
-          <a mat-stroked-button color="primary" feedbackDrawer>Click here</a>
+          <a i18n="Page-informationComponent|Feedback Button@@PageInformationComponentSectionFeedbackButton" mat-stroked-button color="primary" feedbackDrawer>Contact us</a>
+        </span>
+      </mat-card-footer>
+    </mat-card>
+    <!-- user-guide -->
+    <mat-card class="small-card pb-text-align-center pb-margin-05">
+      <mat-card-header class="pb-center-element">
+        <mat-card-title class="pb-icon-60"
+          ><mat-icon color="primary">receipt</mat-icon>
+          <p
+            i18n="Page-informationComponent|Subtitle for User Guide@@PageInformationComponentSubTitleUserGuide"
+          >
+            User Guide
+          </p></mat-card-title
+        >
+      </mat-card-header>
+      <mat-card-content i18n="Page-informationComponent|User Guide Section@@PageInformationComponentSectionUserGuide">
+        <span>
+          If you are new to the Phonebook and want to get to know its features, you can visit the User Guide which is explaining all the features and possibilities.
```

Comment:
Your Sentence is hard to read. Can you change it to something like this "Welcome to Phonebook! 
If you are new to Phonebook and want to know more about its features, you can have a look at the User Guide."

Reply:
Why "Welcome to Phonebook!" ?
The card is at the inforamtion page.

Reply:
it was just an example :) you dont have to use it.

I think this Sentence is just overcomplicated. Go easy, restructure it or spit it up in more easier Sentences